libs/model/src/models/user.ts

```diff
@@ -58,10 +58,10 @@ export default (
     'User',
     {
       id: { type: dataTypes.INTEGER, autoIncrement: true, primaryKey: true },
-      email: { type: dataTypes.STRING, allowNull: true, unique: true },
```

Comment:
Why was unique removed here?

Reply:
incompatible with migrations... uniqueness is enforced by an index, not a constraint